core/src/main/java/org/jdbi/v3/core/statement/AbstractBaseStatement.java

```diff
@@ -23,21 +23,21 @@
 import org.jdbi.v3.core.config.Configurable;
 import org.jdbi.v3.core.generic.GenericTypes;
 
-abstract class BaseStatement<This> implements Closeable, Configurable<This> {
+abstract class AbstractBaseStatement<This> implements Closeable, Configurable<This> {
```

Comment:
I don't love the change adding `Abstract` to all the class names.  Is there a particular reason to do this?

Reply:
Style convention. I won't argue about it if you don't like it.